docs/source/common_issues.rst

```diff
@@ -710,3 +710,44 @@ You can install the latest development version of mypy from source. Clone the
     git clone --recurse-submodules https://github.com/python/mypy.git
     cd mypy
     sudo python3 -m pip install --upgrade .
+
+Differentiate variables and aliases
```

Comment:
Brainstorming: "aliases" is an overloaded term. This might be a slightly clearer title: "Variables vs type aliases" (also similar to the existing section "Invariance vs covariance").